container/Dockerfile

```
@@ -46,14 +42,23 @@ LABEL org.opencontainers.image.authors="searxng <$GIT_URL>" \
       org.opencontainers.image.version="$SEARXNG_GIT_VERSION"
 
 ENV SEARXNG_VERSION="$SEARXNG_GIT_VERSION" \
-    INSTANCE_NAME="SearXNG" \
-    AUTOCOMPLETE="" \
-    BASE_URL="" \
-    BIND_ADDRESS="[::]:8080" \
     SEARXNG_SETTINGS_PATH="$CONFIG_PATH/settings.yml" \
-    UWSGI_SETTINGS_PATH="$CONFIG_PATH/uwsgi.ini" \
-    UWSGI_WORKERS="%k" \
-    UWSGI_THREADS="4"
+    GRANIAN_PROCESS_NAME="searxng" \
+    GRANIAN_INTERFACE="wsgi" \
+    GRANIAN_HOST="::" \
+    GRANIAN_PORT="8080" \
+    # https://github.com/emmett-framework/granian/issues/568
+    GRANIAN_HTTP1_KEEP_ALIVE="false" \
+    GRANIAN_WEBSOCKETS="false" \
+    GRANIAN_LOOP="uvloop" \
+    GRANIAN_BLOCKING_THREADS="2" \
+    GRANIAN_RUNTIME_THREADS="3" \
+    GRANIAN_RUNTIME_BLOCKING_THREADS="2" \
+    GRANIAN_RESPAWN_INTERVAL="0" \
+    GRANIAN_WORKERS_KILL_TIMEOUT="30" \
```

```suggestion
    GRANIAN_WORKERS="1" \
    GRANIAN_WORKERS_KILL_TIMEOUT="30" \
```
I think it makes sense to add this for clarity, so that users can look through the envs here and can simply see how to set up more workers.

---

I would prefer not to make workers option visible, as it induces increasing that value to solve the pains of life by following the bad practices we have now with uWSGI. If the server runs short, increase threads, not workers.

I said it in another issue, but increasing workers doubles the resource usage and reduces the effectiveness of botprotection (placebo?). The option is there documented, you can crank it up whenever you want, but I see this as the *not the best solution* for improving performance.

---

> I would prefer not to make workers option visible, as it induces increasing that value to solve the pains of life by following the bad practices we have now with uWSGI. If the server runs short, increase threads, not workers.
>
In that case, please just mention that in the documentation as well, that's probably a good hint for people running large instances :)

---

8ae618f9949787cf9b7ad8ea023a708bf5ef0a4c